scripts/tests.e2e.sh

```diff
@@ -43,10 +43,27 @@ else
 fi
 
 #################################
-# - Execute in parallel (-p) with the ginkgo cli to minimize execution time.
-#   The test binary by itself isn't capable of running specs in parallel.
+# Determine ginkgo args
+GINKGO_ARGS=""
+if [[ -n "${E2E_SERIAL:-}" ]]; then
+  # Specs will be executed serially. This supports running e2e tests in CI
+  # where parallel execution of tests that start new nodes beyond the
+  # initial set of validators could overload the free tier CI workers.
+  # Forcing serial execution in this test script instead of marking
+  # resource-hungry tests as serial supports executing the test suite faster
+  # on powerful development workstations.
```

Comment:
Might be cleaner to add example usage to the top of the script along with these details vs having to read through the code to understand the script inputs.

Reply:
Sorry if that was not clear I am not questioning the check I am questioning the docs for the ginkgo args. It feels like it should be at the top of the doc maybe above or below here[1]. Just so user doesn't need to sift through the code to understand the params. Jjust an observation.

https://github.com/ava-labs/avalanchego/blob/6e6b19a3909e9aba8e83b3882a5377ef2c9b16ec/scripts/tests.e2e.sh#L5-L7